Dockerfile

```diff
@@ -48,20 +49,18 @@ RUN apt-get update && \
   pkg-config \
   protobuf-compiler \
   python2
-RUN git clone https://github.com/anbox/anbox /anbox
+RUN git clone --recursive https://github.com/anbox/anbox /anbox
 WORKDIR /anbox
 ARG ANBOX_COMMIT
-RUN git pull && git checkout ${ANBOX_COMMIT}
+RUN git pull && git checkout ${ANBOX_COMMIT} && git submodule update --recursive
```

Comment:
You should probably add ``--init`` to the submodule update.

Reply:
Why is that? As far as I'm aware that is only needed if you want to update the `.git/config` file.

Reply:
```
If the submodule is not yet initialized, and you just want to use the setting as stored in .gitmodules, you can automatically initialize the submodule with the --init option.
```

I would add it just to be safe.